src/lib.rs

```diff
@@ -478,7 +478,7 @@ pub struct Options {
     #[clap(long, value_name = "SUDO FLAGS")]
     pub root: Option<Option<String>>,
 
-    /// Sampling frequency in Hz
+    /// Sampling frequency in Hz [default: 997]
```

Comment:
I think we could also set `default = "997"` in the `clap` attribute, which I think would allow changing the type here to `u32` and would probably take care of the docstring for us?

Reply:
I wanted to do that but `fn check` below checks whether the argument is specified explicitly which that would break.